src/pypi_attestations/_impl.py

```diff
@@ -191,6 +191,31 @@ def sign(cls, signer: Signer, dist: Distribution) -> Attestation:
         except ConversionError as e:
             raise AttestationError(str(e))
 
+    @property
+    def claims(self) -> dict[x509.ObjectIdentifier, str]:
+        """Returns the claims present in the Certificate that matches Fulcio OID.
+
+        The complete list is avaible on Fulcio documentation but we only return
+        non deprecated extensions (from 1.3.6.1.4.1.57264.1.8 to .22):
```

Comment:
The docs don't say that `.7` is deprecated ([link](https://github.com/sigstore/fulcio/blob/main/docs/oid-info.md#1361415726417--othername-san))

Reply:
> GitHub Workflow specific OID extensions have been deprecated in favor of provider generic extensions starting from 1.3.6.1.4.1.57264.1.8.

I got confused by this (starting at ...) but you are correct, the 7 has not been deprecated
Great catch !